Library/Homebrew/cmd/info.rb

```diff
@@ -54,7 +55,12 @@ def print_json
     else
       ARGV.formulae
     end
-    json = ff.map(&:to_hash)
+    outdated_formulae = outdated_brews(ff)
+    json = ff.map do |f|
+      hash = f.to_hash
+      hash["outdated"] = outdated_formulae.include?(f)
+      hash
+    end
```

Comment:
I think we should do a deep refactor here by introducing `Formula#outdated?` method.